exercises/Clustering/notebooks/01_Clustering_Embryoid_Body_Data.ipynb

```diff
@@ -754,7 +761,7 @@
     "\n",
     "1. Use `scprep.plot.histogram` (https://scprep.readthedocs.io/en/dev/reference.html#scprep.plot.histogram). \n",
     " * Note, you can pass an list of expression values to get them plotted on the same axis.\n",
-    " * Try using `log='y'` to get a log-scale y-axis\n",
+    " * Try using `log='y'` to get a log-scale y-axis.  This will typically plot the values at the natural log scale.  If you want to different scale, you may have to do one or two additional steps.  **Note Dan/Scott**: Any tips on plotting on log2 scale or log10 for that matter.\n",
```

Comment:
Using `log='y'` doesn't change the actual values, only the scaling of them, so there is actually no 'base' to this log. See the following as an example.
```
import numpy as np
import matplotlib.pyplot as plt

x = np.arange(100)
y = -(x-50)**2 + 50**2 + 1

plt.figure()
plt.plot(x, y)
plt.yscale('log')
plt.title('log scale')

plt.figure()
plt.plot(x, np.log(y))
plt.title('log(y)')

plt.figure()
plt.plot(x, np.log2(y))
plt.title('log2(y)')

plt.figure()
plt.plot(x, np.log10(y))
plt.title('log10(y)')

plt.show()
```

Reply:
I will make the change.  I think the question you asked needs to go in the notebook to clarify why we are scaling the y-axis -- the dynamic range is too big.

Reply:
This is marked as resolved but does not appear to be resolved.

Reply:
Should have been.  Let me check again.